ShippingApp/src/main/java/com/javaschool/service/ShippingServiceImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.javaschool.service;
+
+import com.javaschool.queue.QueueSender;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ShippingServiceImpl implements ShippingService {
+
+    private QueueSender queueSender;
+
+    public ShippingServiceImpl(final QueueSender queueSender) {
+        this.queueSender = queueSender;
+    }
+
+    @Override
+    public List<String> getSize() {
+        List<String> sizes = queueSender.getSize();
+        return sizes;
```

Comment:
why not just `return queueSender.getSize()` ? Look here https://stackoverflow.com/a/31734226

Reply:
It was modified since the variable was not necessary.